packages/semcom-sdk/lib/component/models/component-events.model.ts

```diff
@@ -20,6 +20,16 @@ export interface ComponentReadEventPayload {
    * The uri of the resource to which should be read.
    */
   uri: string;
+
+  /**
+   * The type of data.
+   */
+  type: keyof ComponentDataTypes;
```

Comment:
Maybe also include this as a generic in the interface, to mirror the write and append events?